docs/classes/core.api.html

```diff
@@ -3,12 +3,25 @@
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<title>Api | burstjs</title>
+	<title>Api
+	| burstjs - The Burstcoin Reference Library</title>
 	<meta name="description" content="">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="stylesheet" href="../assets/css/main.css">
+	<link rel="stylesheet" href="../assets/css/custom.css">
 </head>
 <body>
+<a href="https://https://github.com/burst-apps-team/phoenix/tree/master/lib" class="github-corner" aria-label="View source on GitHub">
```

Comment:
develop or master?

Reply:
master... assuming that stable versions are available in master :grin: